Backend/dev/migrations/dynamic-offer-driver-app/0135-add-like-count-in-message.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE atlas_driver_offer_bpp.message_report ADD COLUMN like_status BOOLEAN DEFAULT false;
+ALTER TABLE atlas_driver_offer_bpp.message ADD COLUMN like_count Int DEFAULT 0;
```

Comment:
both should be `NOT NULL`

Reply:
like every message in the beginning  will have 0 like and like_status for that particular message will be false , so if keep it NOT NULL then we have to explicitly assign these value.

Reply:
I have given default value for both the column so there will not be a Null value at any time so what is the need of this constraint ?

Reply:
Why not? anyway it would not worse with constraint

Reply:
understood